src/modules/blocks/tickets/container/style.pcss

```
@@ -37,3 +37,39 @@
 		}
 	}
 }
+
+.tribe-editor__tickets{
+	.tickets-heading {
+		font-size: var( --tec-font-size-7 );
```

```suggestion
		font-size: var(--tec-font-size-7);
```
No white space - & etc (repeated)

---

white spaces were removed.